src/components/vm/consoles/vncBody.jsx

```diff
@@ -34,34 +34,30 @@ export const VncRow = ({ idPrefix, onValueChanged, dialogValues, validationError
 
     return (
         <>
-            <Grid hasGutter md={6}>
-                <GridItem span={6}>
-                    <FormGroup fieldId={`${idPrefix}-address`} label={_("Listening address")}>
-                        <TextInput id={`${idPrefix}-address`}
-                            value={dialogValues.vncAddress}
-                            type="text"
-                            placeholder={_("default")}
-                            onChange={(event) => onValueChanged('vncAddress', event.target.value)} />
-                    </FormGroup>
-                </GridItem>
-                <GridItem span={6}>
-                    <FormGroup fieldId={`${idPrefix}-port`} label={_("Listening port")}>
-                        <TextInput id={`${idPrefix}-port`}
-                            value={dialogValues.vncPort}
-                            type="text"
-                            placeholder={_("automatic")}
-                            validated={validationErrors.vncPort ? "error" : null}
-                            onChange={(event) => onValueChanged('vncPort', event.target.value)} />
-                        { validationErrors.vncPort &&
-                            <FormHelperText>
-                                <HelperText>
-                                    <HelperTextItem variant='error'>{validationErrors.vncPort}</HelperTextItem>
-                                </HelperText>
-                            </FormHelperText>
-                        }
-                    </FormGroup>
-                </GridItem>
-            </Grid>
+            <FormGroup
+                fieldId={`${idPrefix}-portmode`}
+                label={_("Listening")} isInline hasNoPaddingTop isStack>
```

Comment:
This should IMHO be called "Port". "Listening" is a broader concept that spans a list of addresses and such, and I figure in this third "don't allow configuration of VNC listening address" commit we always just set up localhost.

Reply:
Yes, changed.